Services/FileService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LibraryApp.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace LibraryApp.Services
+{
+    internal class FileService : IFileService
+    {
+        private const string FilePath = "books.json";
+        private readonly ILoggerService _logger;
+
+        public FileService()
+        {
+            _logger = new LoggerService();
```

Comment:
I would pass instance of logger service to constructor instead of instantiating it here